start.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+ENCLAVE_ID="wakurtosis"
+PROMETHEUS_YML="prometheus.yml"
+
+# TODO: Check if the enclave already exists 
+kurtosis enclave stop $ENCLAVE_ID
+kurtosis enclave rm $ENCLAVE_ID
+
+# Start the Kurtosis enclave
+kurtosis run ./main.star --enclave-id $ENCLAVE_ID
```

Comment:
Not sure if `./` in `./main.star` here will cause troubles? Maybe remove it just in case? It is a picky thing.

Also, wich `Kurtosis` version are you using? Since in the README.md we are using `Kurtosis version 0.57.4`. The way to pass arguments to Kurtosis changed between versions. 

In 0.57.4 you have to do: `kurtosis run main.star --args '{"key": value}'` for example.
In latest version (0.58.1 atm) you have to do: `kurtosis run main.star '{"name": "Joseph"}'`

We should stablish a version, stick with it, and maybe do a sanity check in your `start.sh` to make sure that the user is using our desired Kurtosis version.

Another example is that the `upload_file` function is called differently between versions (can't check it now since the [changelog](https://kurtosis-tech.github.io/kurtosis/changelog) looks  down).

What do you think, we merge this branch with the current `0.57.4` version, and then we do an update on master where we also do a sanity check for the Kurtosis version? Because there might be too many changes to do the update in this PR.

Reply:
Perfect! Thank you!

Reply:
lets religiously stick to 0.57.4. kurtosis push updates almost daily and are not backward compatible.  disabled the tab-completion stuff $source <(kurtosis completion bash) solely to stop the update nagging. if u r using apt, $apt hold kurtosis-cli will tell the package manager to ignore the updates

Reply:
Yeap, using a Mac with VSC. Let me check what is going on

Reply:
Nope, you commited an empty `start.sh`.

Also, I think it's better to write `kurtosis run ./main.star --enclave-id $ENCLAVE_ID` as `kurtosis run --enclave-id $ENCLAVE_ID main.star `, because it can be missinterpreted as --enclave-id is a parameter for `main.star`, and it is actually a flag for the command `kurtosis run`. Because later we will introduce parameters to `main.star`.

Reply:
Done